css/pwindow.css

```diff
@@ -0,0 +1,207 @@
+.modal {
```

Comment:
- [ ] As you can see in the pictures below the text content is overflowing the container. Please fix the text over flow problem from your modals. 
- [ ] Please remove the scroll behavior of the image inside the modal view. I think you should give its width in `vw` or `%`.


![Screen Shot 2022-07-23 at 20 48 16](https://user-images.githubusercontent.com/12524453/180621012-fff17482-2b29-4124-b838-77bf492c961f.png)